src/widget/geo/geopicker.js

```diff
@@ -1397,16 +1405,20 @@ define( [ 'jquery', 'enketo-js/Widget', 'text!enketo-config', 'leaflet', 'q' ],
         $.fn[ pluginName ] = function( options, event ) {
```

Comment:
Didn't realise this was boilerplate.  Was fiddly getting this working, and difficult to work out what was breaking, hence all the extra logging.  Happy to revert before merging, but would prefer to keep it in until then...